plugins/base/src/main/kotlin/generation/SingleModuleGeneration.kt

```diff
@@ -34,16 +34,16 @@ class SingleModuleGeneration(private val context: DokkaContext) : Generation {
         val documentationModel = mergeDocumentationModels(transformedDocumentationBeforeMerge)
 
         report("Transforming documentation model after merging")
-        val transformedDocumentation = transformDocumentationModelAfterMerge(documentationModel)
+        val transformedDocumentation = documentationModel?.let { transformDocumentationModelAfterMerge(it) }
 
```

Comment:
I'd personally report that nothing to document is left or put the reports in `also` blocks